src/chrome/content/rules/Better-Business-Bureau.xml

```diff
@@ -1,4 +1,8 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://hurdman.app.bbb.org/ => https://hurdman.app.bbb.org/: (6, 'Could not resolve host: hurdman.bbb.org')
+
```

Comment:
If the site doesn't exist anymore, we should remove it from our rulesets. Are there resources below the root that do exist on that hostname?